hcaptcha/res/values-af/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+	<string name="loading_text">Om te verifieer dat jy \'n mens is</string>
```

Comment:
Stripe prefix, also, did this come from our localization pipeline?

Reply:
It did not, this came from the hCaptcha SDK itself. should we defer to our own localization?

Reply:
Hmmm, idk. Let's ask @tillh-stripe